tests/test_categorize.py

```
@@ -31,6 +31,10 @@ def test_swe_abbreviation(self):
         result = categorize_job("SWE Intern - 2025")
         assert result["id"] == "software_engineering"
 
+    def test_swe_abbreviation(self):
```

Ruff (0.15.5)

[error] Redefinition of unused `test_swe_abbreviation` from line 30: `test_swe_abbreviation` redefined here

(F811)